assets/server.ts

```diff
@@ -10,7 +10,9 @@ function options(custom: any = {}) {
 function checkStatus(response: Response): Promise<any> {
     if (response.status === 204) {
         return Promise.resolve({});
-    } else if (response.status >= 200 && response.status < 300) {
+    }else if (response.status == 200 && response.headers.get('Content-Disposition')?.includes('attachment')){
+        return Promise.resolve(response);
```

Comment:
I would probably make this rather a default behaviour, but we need to do that `.json()` call if the response has `content-type` header `application/json`